misc/config_tools/requirements.txt

```diff
@@ -1,5 +1,5 @@
 defusedxml
 lxml
-elementpath==2.5.3
-xmlschema==1.9.2
-tqdm
\ No newline at end of file
+elementpath>=2.5.0
+xmlschema
```

Comment:
Since you removed a version requirement, can you confirm that ANY version of xmlschema is OK?